Vic2ToHoI4/Source/HOI4World/ShipTypes/LegacyShipVariant.cpp

```diff
@@ -0,0 +1,65 @@
+#include "LegacyShipVariant.h"
```

Comment:
this entire file appears to be identical to MtGShipVariants, minus the modules class and operators. Why split it off? Maybe a simple bool would have sufficed if you jammed in into == operator?

Reply:
Legacy also adds upgrades. There's probably a common base class to extract during future cleanup, though.